### DISCUSSION TITLE
Reapply portions of a Server profile

### DISCUSSION THREAD

**Discussion by FIS-rphillips** [HewlettPackard/POSH-HPOneView#592] [Q&A] [answered]
In OneView 6.10 - addressing Synergy blades

Is there a method to reapply certain parts of a Server Profile, specifically the connections?

I'd like to do something like:

Get-ServerProfileTemplate | Get-serverprofile | Foreach-object{
do something here to reapply connections?
}

**Comment by DungKHoang** [comment, marked as answer]
Hi
I believe there is a cmdlet Update-ServerProfile with the attributes you
want to modify
It works for server profile templates too



On Tue, Feb 1, 2022 at 9:11 AM FIS-rphillips ***@***.***>
wrote:

> In OneView 6.10 - addressing Synergy blades
>
> Is there a method to reapply certain parts of a Server Profile,
> specifically the connections?
>
> I'd like to do something like:
>
> Get-ServerProfileTemplate | Get-serverprofile | Foreach-object{
> ?
> }
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/HewlettPackard/POSH-HPEOneView/discussions/592>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGFHKCOMB5ZBPNHPA75CITUZAH2PANCNFSM5NJXZDVA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Dung K Hoang

**Reply by jullienl** [reply]
`Update-OVServerProfile` is used to update Server Profile from Template so it is not the correct cmdlet.

You can use either `Save-OVServerProfile` / `Save-OVServerProfileTemplate` or `Set-OVResource` (which does pretty much the same thing). Basically, you need to run a `get-ovserverprofile` to set an object, then you modify that object with your new settings and run a `Set-OVResource $object`. You can see an example here: [SPT LACP Deployment.ps1](https://github.com/jullienl/HPE-Synergy-OneView-demos/blob/master/Powershell/Compute/SPT%20LACP%20Deployment.ps1)

**Reply by FIS-rphillips** [reply]
Profile is compliant to the template already, and we're not looking to make any new modifications to the server profile or the template. There are no new settings.

We just need to reapply the existing defined connections to the profile.

**Reply by jullienl** [reply]
So in this case, have you tried to use for ex.  `Get-OVServerProfile -Name RHEL-1 | Update-OVServerProfile -connections -Reapply`

**Reply by FIS-rphillips** [reply]
That seems to work out - thanks all for the help!
